boards/arduino-uno/Makefile.features

```diff
@@ -0,0 +1,10 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_gpio
+FEATURES_PROVIDED += periph_spi
+FEATURES_PROVIDED += periph_timer
+FEATURES_PROVIDED += periph_uart
+
+# Various other features (if any)
+
```

Comment:
it seems that this file was introduced (by yourself :) ) in 65d817483b5c3efa58af2e5f2f7a135948b3e664. The file is [here](https://github.com/mali/RIOT/blob/73e02ecc42dff8290a44ecd0709bd5345ca0fa63/boards/arduino-uno/include/arduino_board.h)